src/components/Notification/components/HorizontalSwiper/HorizontalSwiper.ts

```diff
@@ -0,0 +1,195 @@
+export interface Options {
```

Comment:
I think we should mention the source with this url — https://github.com/sciactive/tinygesture/blob/master/src/TinyGesture.ts